### PR TITLE
Fix tests.ps1 to pass the practice tests

### PR DESCRIPTION
**Pull request by valentin-p**
* Pending on the previous PR #888. #888 needs to be merged first to main.
* Pending on the previous PR #884. #884 needs to be merged first to main.
* Update the path /exercises to /exercises/practice so the tests can execute.
* Update the test files to pass the practice exercises the way it was passing before the incorporation of v3.

Out of scope:
* Including the concept exercises in the tests would be done in a later PR, this one is just to rehabilitate the functionality

### CODE REVIEW

**Comment by ErikSchierboom** [comment]
@valentin-p Could you rebase on the latest `main`?

File: exercises/practice/tree-building/Program.fs [hunk 1]
```diff
@@ -13,38 +13,31 @@ open BenchmarkDotNet.Running
...
+                        |> Seq.map (fun m -> m.Descriptor))
+                    |> distinctWithComparer MetricDescriptorEqualityComparer.Instance
+                    |> Seq.map (fun d -> MetricColumn d :> IColumn)
+            }
 
 [<EntryPoint>]
 let main _ =
-    BenchmarkRunner.Run<Benchmarks>(BenchmarkConfig()) |> ignore
+    let  manuelConfig = ManualConfig()
+    let benchmarkConfig = manuelConfig.AddColumnProvider(new AllocatedColumnProvider()).AddDiagnoser(MemoryDiagnoser.Default).AddLogger(ConsoleLogger.Default);
```
Comment (ErikSchierboom) [inline exercises/practice/tree-building/Program.fs:39]:
```suggestion
    let manualConfig = ManualConfig()
    let benchmarkConfig = manualConfig.AddColumnProvider(new AllocatedColumnProvider()).AddDiagnoser(MemoryDiagnoser.Default).AddLogger(ConsoleLogger.Default);
```

**Review by ErikSchierboom — APPROVED** [review]

File: exercises/practice/tree-building/Program.fs [hunk 2]
```diff
@@ -13,38 +13,31 @@ open BenchmarkDotNet.Running
...
-                                r.Metrics.Values
-                                |> Seq.filter (fun m -> m.Descriptor.DisplayName = "Allocated")
-                                |> Seq.map (fun m -> m.Descriptor))
-                            |> distinctWithComparer MetricDescriptorEqualityComparer.Instance
-                            |> Seq.map (fun d -> MetricColumn d :> IColumn)
-                    }
-        }
-
-    do
-        base.Add(columnProvider)
```
Comment (valentin-p) [inline exercises/practice/tree-building/Program.fs:42]: `warning FS0044: This construct is deprecated. This method will soon be removed, please start using .AddColumnProvider()`

**Comment by ErikSchierboom** [comment]
The tests are passing again: https://github.com/exercism/fsharp/pull/887/checks?check_run_id=1813225656 🎉

**Comment by valentin-p** [comment]
image

**Pull request merged**